content/docs/studio/self-hosting/configuration/gitlab.md

```diff
@@ -40,9 +40,12 @@ scmProviders:
 
     clientId: <GitLab OAuth App Client ID>
     secretKey: <GitLab OAuth App Secret Key>
-
-    webhookUrl: <GitLab Webhook URL>
     webhookSecret: <GitLab Webhook Secret>
+
+    # Optional
+    # This is useful in cases where Studio is on an internal
+    # network, but the webhook endpoint is on an external network
+    # webhookUrl: https://webhook.studio.company.com/webhook/gitlab/
```

Comment:
The webhooks will work if the SCM provider has connectivity with Studio. This wouldn't be the case on an internal network, so users can selectively expose the webhook on an external network so that it still works.

Do you think it needs rephrasing to make this more clear?